code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm

```diff
@@ -404,66 +436,173 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 	name = "Tail Trip"
 	action_icon_state = "tail_trip"
 	action_icon = 'icons/Xeno/actions/praetorian.dmi'
-	desc = "Target a marine within two tiles of you to disorient and slows them. Marked enemies receive stronger debuffs and are stunned for a second."
-	ability_cost = 50
+	desc = "Twirl your tail around low to the ground, knocking over and disorienting any adjacent marines. Marked enemies receive stronger debuffs and are briefly stunned."
+	ability_cost = 75
 	cooldown_duration = 8 SECONDS
+	keybind_flags = ABILITY_KEYBIND_USE_ABILITY
 	keybinding_signals = list(
 		KEYBINDING_NORMAL = COMSIG_XENOABILITY_TAIL_TRIP,
 	)
+
+/datum/action/ability/activable/xeno/tail_trip/use_ability(atom/target_atom)
+	. = ..()
+
+	xeno_owner.add_filter("dancer_tail_trip", 2, gauss_blur_filter(1)) //Add cool SFX
+	var/obj/effect/temp_visual/tail_swing/swing = new
+	xeno_owner.vis_contents += swing
+
+	xeno_owner.spin(0.6 SECONDS, 1)
+	xeno_owner.emote("tail")
+	xeno_owner.enable_throw_parry(0.6 SECONDS)
+	playsound(xeno_owner,pick('sound/effects/alien/tail_swipe1.ogg','sound/effects/alien/tail_swipe2.ogg','sound/effects/alien/tail_swipe3.ogg'), 25, 1) //Sound effects
+	xeno_owner.visible_message(span_danger("\The [xeno_owner] sweeps its tail in a low circle!"))
+
+	var/damage = ((xeno_owner.xeno_caste.melee_damage * xeno_owner.xeno_melee_damage_modifier))
+
+	var/list/inrange = orange(1, xeno_owner)
+
+	for (var/mob/living/carbon/human/living_target in inrange)
+		if(living_target.stat == DEAD)
+			continue
+		to_chat(living_target, span_xenowarning("Our legs are struck by \the [xeno_owner]'s tail!"))
+		var/buffed = living_target.has_status_effect(STATUS_EFFECT_DANCER_TAGGED)
+		if(buffed)
+			living_target.ParalyzeNoChain	(2 SECONDS)
+			shake_camera(living_target, 2, 1)
+		living_target.AdjustKnockdown(buffed? 1 SECONDS : 0.5 SECONDS)
+		living_target.adjust_stagger(buffed ? 5 SECONDS : 3 SECONDS)
+		living_target.apply_damage(damage, STAMINA, updating_health = TRUE)
+
+	addtimer(CALLBACK(xeno_owner, TYPE_PROC_REF(/datum, remove_filter), "dancer_tail_trip"), 0.6 SECONDS) //Remove cool SFX
```

Comment:
```suggestion
	addtimer(CALLBACK(xeno_owner, TYPE_PROC_REF(/datum, remove_filter), "dancer_tail_trip"), 0.6 SECONDS)
```

Reply:
I retained the comments from the code I was working off of (defender)
But I can remove it